### PR TITLE
Switching flask_util and django_util to use DictionaryStorage instead…

### PR DESCRIPTION
**Pull request by theacodes**
… of custom classes.

### CODE REVIEW

File: oauth2client/contrib/django_util/storage.py [hunk 1]
```diff
@@ -22,32 +24,4 @@ def get_storage(request):
...
-
-        return credentials
-
-    def locked_put(self, credentials):
-        self.session[_CREDENTIALS_KEY] = credentials.to_json()
-
-    def locked_delete(self):
-        if _CREDENTIALS_KEY in self.session:
-            del self.session[_CREDENTIALS_KEY]
+    return DictionaryStorage(request.session, key=_CREDENTIALS_KEY)
```
Comment (dhermes) [inline oauth2client/contrib/django_util/storage.py:27]: Love it!

File: oauth2client/contrib/flask_util.py [hunk 2]
```diff
@@ -548,31 +547,3 @@ def http(self, *args, **kwargs):
         if not self.credentials:
             raise ValueError('No credentials available.')
         return self.credentials.authorize(httplib2.Http(*args, **kwargs))
-
-
-class FlaskSessionStorage(Storage):
```
Comment (dhermes) [inline oauth2client/contrib/flask_util.py:553]: Good bye, you are the weakest link

**Comment by dhermes** [comment]
LGTM. Coverage drop is fine; math and stuff.

**Comment by nathanielmanistaatgoogle** [comment]
Please rewrite your commit message to conform to http://chris.beams.io/posts/git-commit/#seven-rules?

Change content looks fine.

**Comment by jonparrott** [comment]
@nathanielmanistaatgoogle was your primary issue the tense of the verb? I've fixed that. If you want more detail in the commit message I can add that as well.

**Comment by nathanielmanistaatgoogle** [comment]
"Limit the subject line to 50 characters" is the one I usually notice, and that I noticed here. Notice how the title of the pull request overflows and is truncated with an ellipsis.

**Comment by jonparrott** [comment]
Hmm. I think that errs on the side of rules for the sake of rules, but I shortened it a bit (to 60 characters).

**Comment by nathanielmanistaatgoogle** [comment]
Yeah, I don't like those rules either, but they're where the industry is going and I do favor consistency for metadata like commit messages.

**Comment by jonparrott** [comment]
> Yeah, I don't like those rules either, but they're where the industry is going and I do favor consistency for metadata like commit messages.

Agreed, but sometimes 50 characters isn't enough. :)

Let me know if there's anything else I need to do here. Travis should turn green, the code itself hasn't changed since the PR was opened.

**Comment by nathanielmanistaatgoogle** [comment]
This is fine. Will merge when tests repass.

**Comment by nathanielmanistaatgoogle** [comment]
I don't know what's up with Travis. Merging.

**Comment by jonparrott** [comment]
Thanks!

**Pull request merged**